redis/connection.py

```diff
@@ -607,6 +684,14 @@ def __del__(self):
         except Exception:
             pass
 
+    def _construct_command_packer(self, packer):
+        if packer is not None:
+            return packer
+        elif HIREDIS_PACK_AVAILABLE:
+            return HiredisRespSerializer()
+        else:
+            return PythonRespSerializer(self._buffer_cutoff, self.encoder.encode)
```

Comment:
seems that elif and else are not needed:
```
if packer is not None:
    return packer

if HIREDIS_PACK_AVAILABLE:
    return HiredisRespSerializer()

return PythonRespSerializer(self._buffer_cutoff, self.encoder.encode)
```

Reply:
Technically - yes, we can get rid of `else` but , my understanding of general sentiment regarding `elif ` in Python is: it's cleaner way to show that conditions intended to be mutually exclusive.